src/js/components/Menu/Menu.js

```diff
@@ -172,7 +172,7 @@ class Menu extends Component {
                       }}
                       active={activeItemIndex === index}
                       hoverIndicator="background"
-                      disabled={!item.onClick && !item.href}
+                      disabled={item.disabled || (!item.onClick && !item.href)}
```

Comment:
I'm thinking we should let Button handle the `onClick` and `href` aspects directly and just pass disabled directly through. Would that work?

Reply:
you mean if there is no onClick or href on button to be disabled?

Reply:
If the caller adds `disabled` to the item, then we should pass that to Button. If the caller doesn't pass `disabled` in the item, I don't think Menu should construct another value to pass to Button.

Reply:
ok done! i wonder whether this should be noted as a breaking change. people might have relied on that behavior